dbt_meshify/dbt_projects.py

```diff
@@ -19,17 +20,33 @@ class LocalDbtProject:
     def __init__(self, relative_path_to_project: str) -> None:
         self.relative_path_to_project = relative_path_to_project
         self.manifest = self.get_project_manifest()
+        self.subprojects = []
     
     def path_to_project(self) -> str:
         return os.getcwd() + '/' + self.relative_path_to_project
-    
-    def get_project_manifest(self) -> dict:
+
+    def dbt_operation(self, operation: List[str]) -> dbtRunnerResult:
         start_wd = os.getcwd()
         os.chdir(self.path_to_project())
-        result = dbt_runner.invoke(["--log-level", "none", "parse"])
+        result = dbt_runner.invoke(operation)
         os.chdir(start_wd)
-        if result.success:
-            return result.result
+        return result
+    
+    def get_project_manifest(self) -> Manifest:
+        manifest_result = self.dbt_operation(["--quiet", "parse"])
+        if manifest_result.success:
+            return manifest_result.result
+        
+    def get_subproject_resources(self, subproject_selector: str) -> List[str]:
+        ls_results = self.dbt_operation(["--log-level", "none", "ls", "-s", subproject_selector])
+        if ls_results.success:
+            return ls_results.result
```

Comment:
Non-blocking question: Do we want to raise exceptions returned by the dbtRunner?

Reply:
i would imagine so! not positive how to do that

Reply:
I think something like this would work:

```python
    def get_subproject_resources(self, subproject_selector: str) -> List[str]:
        ls_results = self.dbt_operation(["--log-level", "none", "ls", "-s", subproject_selector])

        if not ls_results.success:
            raise ls_results.exception

        return ls_results.result
```

Given that this would be common code, there's a good argument for having exception handling performed within the `dbt_operation` method.